doc/sphinxman/source/psithoninput.rst

```diff
@@ -108,6 +108,10 @@ a time (and all nodes on the filesystem with |psirc| have similar memory
 capacities), it is advised to set memory in the input file on a
 per-calculation basis.
 
+That same command can be used for PsiAPI mode::
+
+    psi4.set_memory(int(5e8))
```

Comment:
I was thinking it might be wiser to run this through a Python function where users can say `50`, `5e1`, `"30 GB"`, etc and have it parsed correctly. Slightly awkward as is IMO. Ill get on this.